index.html

```diff
@@ -16,26 +16,39 @@
 
   <body>
     <header class="navbar">
-      <div class="navbar-wrapper">
-        <a class="my-logo" href="#">My Logo</a>
-        <nav>
+      <nav class="navbar-wrapper">
+      <div class="my-logo-wrapper"><a class="my-logo" href="#">My Logo</a>
+      </div>
+      <div class="right-menu">
+        <a class="menu-style" href="#work">Portoflio</a>
+        <a class="menu-style" href="#about">About</a>
+        <a class="menu-style" href="#contact">Contact</a>
+        <a href=""><img
+          class="menu-style"
+          src="images/mail.svg"
+          alt="email"
+          /></a>
+        </div>
+      <div class="">
           <img
             class="hamburger-menu"
             src="images/hamburger.svg"
             alt="Hamburger Menu"
           />
+        </div>
         </nav>
-      </div>
+      
     </header>
     <section class="headline">
       <div class="headline-wrapper">
-        <h1 class="greeting-title">Hey There.<br />I’m Peter.</h1>
-        <p class="greeting-description1">I'm a Software Developer</p>
-        <p class="greeting-description2">
-          I can help you build a product , feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don’t hesitate to contact me.
-        </p>
+        <div class="header-greeting"> <h1 class="greeting-title">Hey There.<br />I’m Peter.</h1>
+          <p class="greeting-description1">I'm a Software Developer</p>
+          <p class="greeting-description2">
+            I can help you build a product , feature or website Look through some
+            of my work and experience! If you like what you see and have a project
+            you need coded, don’t hesitate to contact me.
+          </p></div>
+       
```

Comment:
- [ ] To match the Figma design and for the project requirements, kindly make sure you also use the correct text `Font family` to your `greeting-description1` and `greeting-description2` i.e. you should use this `font-family: 'Inter';` to all of them.

- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you also use a `<hx>`(e.g.: h2, h2,...) tag to the text:`I'm a Software Developer` for best semantics tag.